examples/credit-risk-end-to-end/02_Deploying_the_Feature_Store.ipynb

```diff
@@ -0,0 +1,913 @@
+      "10/14/2024 09:03:34 PM root WARNING: _list_feature_views will make breaking changes. Please use _list_batch_feature_views instead. _list_feature_views will behave like _list_all_feature_views in the future.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# curl command to online server to get data from the online store\n",
+    "cmd = \"\"\"http://localhost:6566/get-online-features \\\n",
+    "    -d '{ \n",
+    "            \"feature_service\": \"loan_fs\",\n",
+    "            \"entities\": {\"ID\": [18, 764]}\n",
+    "        }'\n",
+    "\"\"\"\n",
+    "\n",
+    "response = !curl -X POST {cmd}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 37,
+   "id": "c8b3ea8d-1270-44f2-9a5c-1d8b54b36b2b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current',\n",
+       " '                                 Dload  Upload   Total   Spent    Left  Speed',\n",
+       " '',\n",
+       " '  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0',\n",
+       " '100  1235  100  1139  100    96   392k  33874 --:--:-- --:--:-- --:--:--  603k',\n",
+       " '{\"metadata\":{\"feature_names\":[\"ID\",\"credit_amount\",\"installment_commitment\",\"duration\",\"num_dependents\",\"existing_credits\",\"residence_since\",\"age\"]},\"results\":[{\"values\":[18,764],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"1970-01-01T00:00:00Z\",\"1970-01-01T00:00:00Z\"]},{\"values\":[12579.0,2463.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[4.0,4.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[24.0,24.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[1.0,1.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[1.0,2.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[2.0,3.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]},{\"values\":[44.0,27.0],\"statuses\":[\"PRESENT\",\"PRESENT\"],\"event_timestamps\":[\"2023-10-04T11:18:21Z\",\"2023-10-05T07:35:12Z\"]}]}']"
+      ]
+     },
+     "execution_count": 37,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "b9b829ec-2335-4bba-a3df-7d18f766f758",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d2849a99-0193-4f6f-8fb5-3c667c091499",
+   "metadata": {},
+   "source": [
+    "### Server Shutdown"
```

Comment:
I m also divided, WDYT of moving this section to separate notebook and keep at the end. short link here. so that if someone is running the entire jupyter notebook cells online/offline server won't shutdown.

Reply:
Hmm. I kind of like this idea. Maybe a "Cleanup" notebook.  With commands to shutdown the servers and get rid of the files that were created. As it is, it could be a little hard to remember to go back to notebook 2 and shutdown.

Reply:
I've created a cleanup notebook. It ended up being a little more complicated than I expected, in particular for cleaning up the server processes. Shutting down the processes in a separate notebook was leading to leftover zombie procs. To remedy this, I had to change the commands for setting up the servers slightly.